pycomlink/processing/nearby_rain_retrival.py

```diff
@@ -31,31 +36,29 @@ def nearby_determine_reference_level(wet, pmin, pmax=None):
 
     dry = (wet == 0).where(~np.isnan(wet))
 
-    if pmax is not None:
-        pmean = ((pmin + pmax) / 2).where(dry == 1)
-    else:
-        pmean = pmin.where(dry == 1)
+    pmean = ((pmin + pmax) / 2).where(dry == 1)
 
-    pref = pmean.rolling(time=96, min_periods=1).median()
+    pref = pmean.rolling(time=n_average_dry, min_periods=1).median()
 
     return pref
 
 
-def nearby_correct_recieved_signals(pmin, wet, pref, pmax=None):
+def nearby_correct_received_signals(pmin, pmax, wet, pref):
     """
-    Determine reference/baseline level during rain events.
+    Correcting pmin and pmax to prevent rainfall estimation during dry time
```

Comment:
Change the phrasing to
```
Correcting pmin and pmax so that no rainfall estimation is carried out
during dry time steps. All time steps of pmin which are not classified wet
and pmin is smaller than pref are set to pref. Similarly, all time steps of
pmax where either the corrected pmin (p_c_min) is not smaller than pref or
pmax is not smaller than pref are set to pref. This ensures that only wet
time steps are used for rainfall estimation an and that pmax is not above
pref  which would lead to an overestimation of rainfall. 
```